### ISSUE TITLE
Good work.

### ISSUE THREAD

**Issue by mopplayer** [smartperson/spartan-edge-accelerator-graphical-system#16]
Hi, thanks to your great work.

However, the seeed studio abandoned this board for a long time.
And this board is mainly to do like a shield board.

An observation from the schematic design:
ESP32 I/Os are limited to arduino I/O, there is no more I/O to use.
And also, FPGA I/Os are limited to arduino I/O, there are only 10 I/Os to use (IO0~IO9).

If I do not use arduino,  QSPI communicates between ESP32 and FPGA, which is the major application.

Standalone mode, the FPGA is limited to HDMI/Camera/ADC /DAC....

After seeing your idea, I do not have ideas to do something interesting using ONLY 10 I/Os.

A SDRAM needs 3X I/Os.

Yeah, the low cost and good Spartan-7 chip, but design is so limited to do few things.

**Comment by smartperson** [comment]
True that Seeed isn't doing much of anything with this board anymore. I wonder if some of it is related to the general supply chain issues and chip shortages that we've been having for the last few years.

Beyond that, I think that sometimes it's more fun to work in a platform where there _are_ limitations. It forces you to be creative, and I think the overall low cost of the board makes it a lot of fun to play with. Do you think there's another board that includes a digital video output that is a good platform to play with?

I have an application I'm working with that runs on the ESP32 side of the board, and it's able to communicate with the Spartan 7. I like the idea of figuring out a hardware implementation on the FPGA that pairs with software that runs on the ESP32. Other than moving to the Zynq, there aren't too many affordable platforms that offer this kind of combination.

[/smartperson/spartan-edge-accelerator-arduino-qspi](https://github.com/smartperson/spartan-edge-accelerator-arduino-qspi)

**Comment by mopplayer** [comment]
Yeah, the board is in stock in a short period, then out of stock.
The reason is a common issue to the China boards, so that's why it's so cheap.

The CHIPS Act from the U.S. government will cause a more worse shortage in China. Because a Spartan-7 chip is created in a 28 nm process. Moreover, AMD-Xilinx is more concerned about high grade FPGAs, there are a few supplies for lowend grade FPGAs.

The weakness of this board, which is exactly used as an Arduino shield.

If you are not restrict to Xilinx FPGA, then Sipeed Tang Nano 4K (low to $13) is also a good choice, which features a GOWIN FPGA included Cortex-M3 hardcore and a HDMI port.

Might you also give it a try.
